tests/jobs.py

```diff
@@ -1662,3 +1662,74 @@ def test_remove_file_clears_import_error(self):
         import_errors = session.query(models.ImportError).all()
 
         self.assertEqual(len(import_errors), 0)
+
+    def test_reset_orphaned_tasks(self):
+        prefix =  'scheduler_job_test_test_reset_orphaned_tasks'
```

Comment:
Seems like there could be some more test coverage here (e.g. the noop case where no orphans exist, the "all" dagruns case when no dagruns exist)